Data/Localizations/ru.lua

```diff
@@ -1,3 +1,6 @@
+-- I marked every line I edit with *-- KartonArmadon* sign. This is for easier work in the future
```

Comment:
I somewhat understand your reasoning behind this, but it's not something that should find its way to a production deployment.

For now there should be no problem keeping these on your end.

I will add your name to the russian translation credits of course!

Reply:
Np! I leave this in my local changes :)